### PR TITLE
Feat/class weights for classification dict

### PR DESCRIPTION
**Pull request by javispp**

### CODE REVIEW

File: src/biome/text/modules/heads/classification/classification.py [hunk 1]
```diff
@@ -39,8 +43,19 @@ def __init__(
         self._multilabel = multilabel
 
         # metrics and loss
+        if class_weights != None:
```
Comment (dcfidalgo) [inline src/biome/text/modules/heads/classification/classification.py:46]: With the type checks that follow, this if statement is superfluous.

File: src/biome/text/modules/heads/classification/classification.py [hunk 2]
```diff
@@ -39,8 +43,19 @@ def __init__(
...
+            if isinstance(class_weights, list):
+                class_weights = torch.FloatTensor(class_weights)
+            elif isinstance(class_weights, dict):
+                classes = [
+                    vocabulary.label_for_index(self.backbone.vocab, i)
+                    for i in range(len(self.labels))
+                ]
+                class_weights = torch.FloatTensor(
+                    [class_weights[label] for label in classes]
+                )
```
Comment (dcfidalgo) [inline src/biome/text/modules/heads/classification/classification.py:56]: Much easier, just:
```suggestion
                class_weights = torch.tensor(
                    [class_weights[label] for label in labels]
                )
```

File: src/biome/text/modules/heads/classification/classification.py [hunk 3]
```diff
@@ -39,8 +43,19 @@ def __init__(
         self._multilabel = multilabel
 
         # metrics and loss
+        if class_weights != None:
+            if isinstance(class_weights, list):
+                class_weights = torch.FloatTensor(class_weights)
```
Comment (dcfidalgo) [inline src/biome/text/modules/heads/classification/classification.py:48]: Better to use:
```suggestion
                class_weights = torch.tensor(class_weights)
```
to automatically cover ints.

**Review by dcfidalgo — CHANGES_REQUESTED** [review]
Nice, thanks! Just a few inline comments. I will close the other PR then.

**Review by dcfidalgo — APPROVED** [review]

**Comment by dcfidalgo** [comment]
Great, thanks a lot @javispp In the end i changed the parameter name to `label_weights` to make it more consistent regarding the `labels` parameter. I also added the parameter to the other classification heads and completed the doc strings.
